packages/core/deploy/036_deploy_across_ovm_bridge_deposit_box.js

```diff
@@ -0,0 +1,30 @@
+// This deploy script should be run on an optimism provider.
+const { getAddress } = require("@uma/contracts-node");
+const { ZERO_ADDRESS } = require("@uma/common");
+const func = async function (hre) {
+  const chainId = await hre.web3.eth.net.getId();
+
+  const { deployments, getNamedAccounts } = hre;
+  const { deploy } = deployments;
+
+  const { deployer } = await getNamedAccounts();
+
+  // Map L2 chain IDs to L1 chain IDs to find associated bridgeAdmin addresses for a given L2 chain ID.
+  const l2ChainIdToL1 = {
+    69: 42, // optimism testnet -> kovan
+    10: 1, // optimism mainnet -> mainnet
+  };
+
+  const bridgeAdminAddress = await getAddress("BridgeAdmin", l2ChainIdToL1[chainId]);
```

Comment:
This may be a little hacky, but maybe the simplest way to do this without having to change the way we do migrations _and_ have this contract _always_ be available is that if this deploy script is called on a chain without a pre-existing mapping, we just set it to map to _its_ own BridgeAdmin so we have the full ecosystem operating on a single chain.

```js
const bridgeAdminAddress = l2ChainIdToL1[chainId] ? await getAddress("BridgeAdmin", l2ChainIdToL1[chainId]): (await deployments.get("BridgeAdmin")).address;
```

Thoughts?

Reply:
sure. that sounds reasonable.